manim_slides/present/__init__.py

```diff
@@ -202,7 +202,7 @@ def str_to_int_or_none(value: str) -> Optional[int]:
     "screen_number",
     metavar="NUMBER",
     type=int,
-    default=None,
+    default=0,
```

Comment:
The initial motivation behind an `Optional[int]` was to differentiate `None` from any provided value. `None` would the mean: display the Windows at the default place, chosen by Qt, which is usually the current screen in "focus". Now that you default to `0`, it will always put the window to the screen number 0, regardless of the current screen that is used. You can see this by launching `manim-slides` from a terminal. Depending on where the terminal is (i.e., screen 0 or screen 1), the Manim Slides application will be put on the same screen. Another difference is (currently) that setting the screen number puts the window in the top left corner, which is not perfect. This could be fixed.

I think I prefer to keep the default value to `None`. Then, we need some way to get the primary screen (e.g., https://stackoverflow.com/a/74418148), and use it as default if no screen was provided.

Reply:
I will update this soon, but was wondering if it would be better to have the windows maximized by default (not full screen, just maximized). Before this PR, the presentation would always take up most of the screen, but I would still have to click maximize to make it take up all but the taskbar. This would also fix the window only being created on top-left (although setting the screen to None would have a similar effect). lt would still be possible to minimize/adjust the window. Either way, I will implement the change soon.

Reply:
Good point, but I think that maximizing the window makes little sense, mainly because of the top bar that prevents the common 16/9 ratio to be kept, but also since people may want to launch a smaller window just for debugging, as I do when I render in lower resolution.

The nice thing about not maximizing the window is that Qt will then automatically pick an appropriate size.

For real presentations, then you use full screen.

Reply:
I found that I could just leave both screens as None, and Qt wouldn't put the two windows on the same screen. It kind of makes me wonder why I had implemented the check beforehand. Let me know if you want me to implement the check just for redundancy purposes.